src/renderers/index.ts

```diff
@@ -118,20 +122,14 @@ export
 class ConsoleTextRenderer implements IRenderer<Widget> {
   mimetypes = ['application/vnd.jupyter.console-text'];
 
-  constructor() {
-    this._converter = new Convert({
-      escapeXML: true,
-      newline: true
-    });
-  }
-
   render(mimetype: string, data: string): Widget {
     let w = new Widget();
-    w.node.innerHTML = this._converter.toHtml(data);
+    let el = document.createElement('pre');
+    let esc = escape_for_html(data);
+    el.innerHTML = ansi_to_html(esc);
```

Comment:
Can we use the `ansi_to_html(esc, {use_classes: true})` option, so we can easily style the colors?